src/runtime-helpers.js

```diff
@@ -649,25 +706,19 @@ function processPackage (packages, deploymentPackages, deploymentTriggers, param
     }
 
     if (packages[key]['apis']) {
-      Object.keys(packages[key]['apis']).forEach((api) => {
-        if (packages[key]['apis'][api]) {
-          const objectAPI = createApiObject(packages, key, api, ruleAction, arrSequence, namesOnly)
-          objectAPI.name = api
-          apis.push(objectAPI)
-        } else {
-          throw new Error('Arguments to create API not provided')
-        }
+      Object.keys(packages[key]['apis']).forEach((apiName) => {
+        const apiRoutes = createApiRoutes(packages[key], key, apiName, ruleAction, arrSequence, namesOnly)
+        routes = routes.concat(apiRoutes)
       })
     }
   })
-  const entities = {
-    pkgAndDeps: pkgAndDeps,
-    apis: apis,
-    triggers: triggers,
-    rules: rules,
-    actions: actions
+  return {
+    pkgAndDeps,
+    apis: routes,
```

Comment:
We could stick to `apis` for consistency reasons or is there a reason to name it `routes` that I'm missing?

Reply:
@basecode because it is not apis but routes (flattened route definition), so i am having a hard time naming it differently :)